_posts/2016-06-15-git-merge-conflict-strategies.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "Tools and strategies for resolving Git merge conflicts"
+date: 2016-06-15
+author: Dan Murphy
+tags: git
+summary: An overview of tools and strategies I like to use when I encounter conflicts during merges.
+featured_image: "/assets/img/blog/git_merge_conflict_screenshot.png"
+featured_image_alt: "screenshot of git merge output"
+---
+
+If you use Git for version control and you collaborate frequently, then you are bound to run into merge conflicts from time to time.
+
+In an ideal world you merge upstream updates into your work frequently, and the conflicts are relatively minor and straightforward to resolve.
```

Comment:
maybe another half-sentence to clarify what "upstream updates" mean. Something along the lines of "changes that make their way to production with your parallel feature work" ... or something like that.